src/ReplyEditor.res

```diff
@@ -41,8 +41,7 @@ let make = (
             }
           | None => []
           }
-          
-          onReplySend(conversation, self.state.message_text, attachments)
+          onReplySend(conversation, state.message_text, attachments)
```

Comment:
This seems like the wrong fix; the problem is actually on line 35, where message_text is always assigned to an empty string.

I had this on my local branch but didn't have the time to commit it yet, will fix that later.

On another note, this may or may not work by accident, since `self` is referring to the most recent `state` after the update has been executed.

Reply:
actually, now trying it out again, this is the correct way of handling it for this case. I thought line 35 was faulty, but the idea is to clear the text input after submitting.